New article: managing contacts

sbastn opened the pull request:
This PR introduces a new article that explains how to manage a contact. It explains the importance of a having correct contact info and well as how to create, edit, and delete a contact.

Closes #162

aeden commented:
@sbastn what do you think of renaming the file to `contact-management.markdown` so it is grouped together with other contact-related articles?

aeden commented:
Also, what do you think about adding a "Contacts" item to the left-hand menu and including this and the other contact articles in that section?

sbastn commented:
Thanks for the feedback @aeden! I have added a new category called "Contacts" and added a couple of articles that I think are contact related. I didn't want to pull in the WHOIS articles since we already have a category for them.

<img width="571" alt="screen shot 2017-10-03 at 1 41 18 pm" src="https://user-images.githubusercontent.com/80610/31123518-902162ba-a840-11e7-9c21-1853da23b24d.png">

weppos commented on content/articles/contact-management.markdown:
```diff
@@ -0,0 +1,51 @@
+
+If you wish to update a specific domain only, you can either [replace the contact entirely](/article/changing-domain-contact/#replacing-a-domain-contact) with a new one, or [update the contact](/articles/changing-domain-contact/#updating-a-domain-contact) like we just mentioned to make that change.
+
+### Contact lock
```
I would change this title to:

    Domain registrant change and ICANN transfer lock
